### PR TITLE
Update k8s network model section

### LINKED ISSUE #44246: Update the Services, Load Balancing, and Networking index page
Page to be updated: https://kubernetes.io/docs/concepts/services-networking/

What needs to be done?

- [The Kubernetes network model](https://kubernetes.io/docs/concepts/services-networking/#the-kubernetes-network-model) section needs to be updated per [this comment](https://github.com/kubernetes/website/pull/32363#discussion_r1413838618) and [this comment](https://github.com/kubernetes/website/pull/32363#discussion_r1413840655)
- For technical accuracy and/or reference, you may also check PR #28454 

/sig network
/help
/good-first-issue

cc: @aojea

### PR DESCRIPTION
**Pull request by abhilasha2418**
Closes #44246 

- **Page to be updated**: https://kubernetes.io/docs/concepts/services-networking/

- **changes** : [The Kubernetes network model](https://kubernetes.io/docs/concepts/services-networking/#the-kubernetes-network-model) section updated per https://github.com/kubernetes/website/pull/32363#discussion_r1413838618 and https://github.com/kubernetes/website/pull/32363#discussion_r1413840655

### CODE REVIEW

File: content/en/docs/concepts/services-networking/_index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: >
 
 ## The Kubernetes network model
 
-Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address. 
+Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address(one address per IP address family).
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/_index.md:10]:
```suggestion
Every [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its own unique cluster-wide IP address
(one address per IP address family).
```

**Review by sftim — COMMENTED** [review]
Thanks

One nit

File: content/en/docs/concepts/services-networking/_index.md (hunk 1 above)
Comment (abhilasha2418) [reply content/en/docs/concepts/services-networking/_index.md]: Done!

**Review by sftim — COMMENTED** [review]
Thanks for the tweak

/lgtm
/approve

**Pull request merged**